frontend/packages/dev-console/integration-tests/features/Knative/eventing-kamletes.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Creation of Kamelets via Event Source Creation
+    As a user, I should be able to install and use more camel connectors
+
+
+    Background:
+        Given user has installed OpenShift Serverless Operator
+        And user has created Knative Serving and Knative Eventing CR
+        And user has installed Red Hat Integration - Camel K Operator
+        And user has created Integration Platform CR
+        And user has selected "aut-test-kamelets" namespace
```

Comment:
```suggestion
        And user has selected "aut-test-kamelets" namespace
        And user has created Integration Platform CR
```

should it be like this i.e ns should be selected before creating IP?

Reply:
It's correct Jai. For the statetment `user has created Integration Platform CR`, scenario from operators.feature will run in the background.

Reply:
which ns user is in when `user has created Integration Platform CR` ??

Reply:
`"aut-test-kamelets"`

Reply:
so as you described this happens here https://github.com/openshift/console/pull/7441/files#diff-072029627bf4da5d30ef8de570b1ac4f646eb59ffd5f2a2f2516db68135c256aR141 so adding it here again may not be needed

```
 And user has selected "aut-test-kamelets" namespace
```